packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -279,6 +281,10 @@ class ReactTestInstance {
     return this._currentFiber().memoizedProps;
   }
 
+  get treeBaseTime(): number {
```

Comment:
The Devtools are going to use a private api to access this data in a way that we support only because we can push new versions of it so mitigates maintainance burden.

You can use the private api to test this behavior since the private api is the “internally public” api, the one actually used.

Even if we wanted to expose this in the future I don’t think we are ready to do that yet.

Reply:
Sure, that makes sense.